frontend/src/hooks/useSuggestedLabels.js

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { PROJECT_ID } from "../store";
+import { getConfig } from "../utils/fetch_configs";
+
+const useSuggestedLabels = (text) =>
+    useQuery({
+        queryKey: ["suggestedLabels", PROJECT_ID],
+        queryFn: () =>
+            fetch(`/api/comparisons/${PROJECT_ID}?text=${text}`, getConfig())
```

Comment:
I'm getting an error when I go to the history table now for the default project (like the three users one) and open one of the data items with the >

Even though the project has only a few labels, it looks like it's trying to load label suggestions
<img width="1026" alt="Screen Shot 2023-05-25 at 11 50 43 AM" src="https://github.com/RTIInternational/SMART/assets/10951569/42172e86-a51b-4769-94e7-81a2e8c0fec8">

At the same time we see the following errors on the backend (I think we've seen these before?)
```
[25/May/2023 15:51:22] "GET /api/comparisons/2/?text=10%20weeks%20old%20and%20learning%20%F0%9F%98%8D HTTP/1.1" 500 16657
backend              | Internal Server Error: /api/comparisons/2/
backend              | Traceback (most recent call last):
backend              |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 47, in inner
backend              |     response = get_response(request)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 181, in _get_response
backend              |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
backend              |     return view_func(*args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/django/views/generic/base.py", line 70, in view
backend              |     return self.dispatch(request, *args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 509, in dispatch
backend              |     response = self.handle_exception(exc)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 469, in handle_exception
backend              |     self.raise_uncaught_exception(exc)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
backend              |     raise exc
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/views.py", line 506, in dispatch
backend              |     response = handler(request, *args, **kwargs)
backend              |   File "/usr/local/lib/python3.8/site-packages/rest_framework/decorators.py", line 50, in handler
backend              |     return func(*args, **kwargs)
backend              |   File "/code/core/views/api_annotate.py", line 679, in embeddings_comparison
backend              |     cosine_scores = util.pytorch_cos_sim(text_embedding, project_labels_embeddings)
backend              |   File "/usr/local/lib/python3.8/site-packages/sentence_transformers/util.py", line 21, in pytorch_cos_sim
backend              |     return cos_sim(a, b)
backend              |   File "/usr/local/lib/python3.8/site-packages/sentence_transformers/util.py", line 42, in cos_sim
backend              |     return torch.mm(a_norm, b_norm.transpose(0, 1))
backend              | RuntimeError: mat1 and mat2 shapes cannot be multiplied (1x384 and 0x1)
```